module/documents/active-effect.mjs

```diff
@@ -364,15 +365,30 @@ export default class ActiveEffect5e extends ActiveEffect {
     const actor = app.object.actor;
     const level = foundry.utils.getProperty(actor, "system.attributes.exhaustion");
     if ( Number.isFinite(level) && (level > 0) ) {
+      const img = this._getExhaustionImage(level);
```

Comment:
Since this method is a hook handler, `this` is the `Hooks` singleton rather than `ActiveEffect5e`, so we'll need to manually refer to that.

```suggestion
      const img = ActiveEffect5e._getExhaustionImage(level);
```